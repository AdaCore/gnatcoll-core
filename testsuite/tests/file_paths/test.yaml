description: Extensive test for the GNATCOLL.File_Paths API
driver: build_run_diff
