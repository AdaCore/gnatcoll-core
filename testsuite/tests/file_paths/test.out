"", ':', IF_EMPTY
=================

"bar": abs("bar")
"foo": ""
"dir1/bar": ""
"dir1/foo": abs("dir1/foo")
"dir2/foo": abs("dir2/foo")
abs("dir1/foo"): abs("dir1/foo")

"dir1|dir2", '|', IF_EMPTY
==========================

"bar": abs("dir2/bar")
"foo": abs("dir1/foo")
"dir1/bar": ""
"dir1/foo": ""
"dir2/foo": ""
abs("dir1/foo"): abs("dir1/foo")

"dir2|dir1", '|', IF_EMPTY
==========================

"bar": abs("dir2/bar")
"foo": abs("dir2/foo")
"dir1/bar": ""
"dir1/foo": ""
"dir2/foo": ""
abs("dir1/foo"): abs("dir1/foo")

"dir2", '|', CWD_FIRST
======================

"bar": abs("bar")
"foo": abs("dir2/foo")
"dir1/bar": ""
"dir1/foo": abs("dir1/foo")
"dir2/foo": abs("dir2/foo")
abs("dir1/foo"): abs("dir1/foo")

"dir2", '|', CWD_LAST
=====================

"bar": abs("dir2/bar")
"foo": abs("dir2/foo")
"dir1/bar": ""
"dir1/foo": abs("dir1/foo")
"dir2/foo": abs("dir2/foo")
abs("dir1/foo"): abs("dir1/foo")

Done.
